api/src/app.py

```diff
@@ -191,3 +193,18 @@ def robots() -> str:
         Allow: /docs
         Disallow: /
         """
+
+
+def register_well_known(app: APIFlask) -> None:
+    @app.route("/.well-known/pki-validation/<file_name>")
+    @app.doc(hide=True)
+    def get_dv_verification_content(file_name: str) -> tuple:
+        try:
+            with open(
+                f"{Path(__file__).resolve().parent}/static/domain_verification_assets/{file_name}"
```

Comment:
I feel like, as written, this can easily expose and random file on our server

Like so:

```
curl api.simpler.grants.gov/.well-known/pki-validation/../../../../secret-file
```

Reply:
Assuming this route needs to be shaped this way, and take in a file/key name, I'd probably aim to make a mapping of keys to something loaded from an env var.